src/common/low_precision_transformations/include/low_precision/cleanup_transformation.hpp

```diff
@@ -17,7 +17,7 @@ namespace low_precision {
 class LP_TRANSFORMATIONS_API CleanupTransformation : public LayerTransformation {
 public:
     CleanupTransformation(const Params& params);
-    virtual ~CleanupTransformation() = default;
+    virtual ~CleanupTransformation();
```

Comment:
Maybe this dtor can be removed and move to cpp file destructor of base class `ov::pass::PassBase` ?
IF that will work all "virtual" default destructor can be removed in children classes.

Reply:
Such implicit virtual dtor is in [MatcherPass](https://github.com/openvinotoolkit/openvino/blob/d757efd7fb3415a3dbda10941b3dae0ace0ac16e/src/core/include/openvino/pass/matcher_pass.hpp#L54) definition.

Reply:
removed explicit default destructors definitions for derived classes